index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width" />
+
+    <meta
+      name="description"
+      content="Author: Amedzro Emmanuel Elikplim, School: Microverse, Project: Portfolio"
+    />
+
+    <link rel="stylesheet" href="styles.css" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      rel="preload"
+      as="style"
+      onload="this.onload = null; this.rel = 'stylesheet'"
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;500;700&family=Roboto&display=swap"
+    />
+
+    <link
+      rel="preload"
+      as="style"
+      onload="this.onload = null; this.rel = 'stylesheet'"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+
+    <title>Portfolio</title>
+  </head>
+
+  <body>
+    <header>
+      <nav class="toolbar">
+        <ul class="toolbar-items">
+          <li><h3 id="logo-text">John Doe</h3></li>
```

Comment:
### [OPTIONAL]

- [ ] One of the requirements says that your logo should be a link. Kindly I would suggest putting this logo in a `<a></a>` tag

Reply:
Modify the logo to an `<a>` tag